giraffe/src/transforms/band.ts

```diff
@@ -126,6 +125,77 @@ export const getBandName = (fillColumnMap: ColumnGroupMap): string => {
   }, '')
 }
 
+export const alignMinMaxWithBand = (
+  lineData: LineData,
+  bandIndexMap: BandIndexMap
+): LineData => {
+  const {rowIndices, maxIndices, minIndices} = bandIndexMap
+
+  for (let i = 0; i < rowIndices.length; i += 1) {
```

Comment:
there's a whole lotta variable redeclaration and function creation in these loops here. is the `rowIndicies` number expected to stay small?